### PR TITLE
Add combobox and use for pronouns, location

### PR DESCRIPTION
**Pull request by hidde**
This wraps the [combobo](https://github.com/dequelabs/combobo) library to create a combobox component

### CODE REVIEW

File: src/components/Combobox.vue [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+      this.combobo.addEventListener('selection', () => {
+        this.$nextTick(() => {
+          this.$emit('input', this.$refs.comboboxInput.value);
+        });
+      });
+    },
+    checkInput(e) {
+      const alwaysAllowed = [9, 13, 27, 16, 8]; // ignore tab, enter, escape, shift, backspace
+
+      if (alwaysAllowed.indexOf(e.which) >= 0) {
```
Comment (Gregoor) [inline src/components/Combobox.vue]: [`array.includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) would be more explicit.

Also `indexOf` return `0` if the first element matches, so checking for `>= 0` might not be what you want here.

File: src/components/Combobox.vue [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    source: Array,
+    value: String,
+    allowCustomInput: {
+      type: Boolean,
+      default: true,
+    },
+    placeholder: String,
+  },
+  methods: {
+    init() {
```
Comment (Gregoor) [inline src/components/Combobox.vue]: I'd personally just move that to `mounted()` to reduce indirection.

File: src/components/Combobox.vue [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+      this.combobo = new Combobo({
+        input: comboboxInput,
+        list: comboboxList,
+        options: this.optionEls,
+        activeClass: 'combobox__option--selected',
+      });
+
+      this.combobo.addEventListener('selection', () => {
+        this.$nextTick(() => {
```
Comment (Gregoor) [inline src/components/Combobox.vue]: Why `$nextTick`?

File: src/components/Combobox.vue (hunk 1 above)
Comment (hidde) [reply src/components/Combobox.vue]: Hm, `array.includes` reads better, but does not work in IE11, not sure if we want to polyfill, I'll double check our browser requirements.

File: src/components/Combobox.vue (hunk 2 above)
Comment (hidde) [reply src/components/Combobox.vue]: Haha, I was probably prematurely optimising, not sure if we'll meet a case where we'd want to init on a different lifecycle hook. Have moved to `mounted()` now.

File: src/components/Combobox.vue (hunk 3 above)
Comment (hidde) [reply src/components/Combobox.vue]: For some reason, I was unable to emit the input event on `selection`, waiting for the next tick (DOM update) fixed that.

File: src/components/Combobox.vue (hunk 1 above)
Comment (hidde) [reply src/components/Combobox.vue]: Doublechecked, IE11 is not a requirement, `array.includes` it is!

**Pull request merged**